### PR TITLE
Coinflip and FloatArgument

### PR DESCRIPTION
**Pull request by 0mlml**

### CODE REVIEW

File: src/main/java/dev/mlml/command/Context.java [hunk 1]
```diff
@@ -41,7 +44,7 @@ public Context(Message message) {
         this.member = message.getMember();
         this.gChannel = message.getGuildChannel();
         this.channel = message.getChannel();
-
+        this.guild = message.getGuild();
```
Comment (RAdu2005) [inline src/main/java/dev/mlml/command/Context.java:47]: Maybe refactor this to the rest of the codebase

File: src/main/java/dev/mlml/command/argument/FloatArgument.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package dev.mlml.command.argument;
+
+public class FloatArgument extends ArgumentBase<Float> {
+    public FloatArgument(String name, String description, boolean isRequired) {
+        super(name, description, isRequired);
+    }
+
+    @Override
+    public Float parse(String input) {
+        return Float.parseFloat(input);
```
Comment (RAdu2005) [inline src/main/java/dev/mlml/command/argument/FloatArgument.java:10]: Is there error handling/a way of preventing abuse on this?

**Review by RAdu2005 — CHANGES_REQUESTED** [review]

File: src/main/java/dev/mlml/command/Context.java (hunk 1 above)
Comment (0mlml) [reply src/main/java/dev/mlml/command/Context.java:47]: There's no other uses of this, thought `Message.getGuildId()` is used elsewhere.

File: src/main/java/dev/mlml/command/argument/FloatArgument.java (hunk 2 above)
Comment (0mlml) [reply src/main/java/dev/mlml/command/argument/FloatArgument.java:10]: Fixed in new commit

**Review by RAdu2005 — APPROVED** [review]

**Pull request merged**